tags/environmentalsustainability.md

```diff
@@ -0,0 +1,77 @@
+# CNCF TAG Environmental Sustainability
+
+## Introduction
+
+The Paris Climate Accords outline the link between economic activity, greenhouse gas (GHG) emissions, and the impacts of climate change. GHG  emissions, primarily made from Carbon Dioxide, are released during the combustion of fossil fuels to produce electricity. To reach net zero goals, we need to identify how to reduce the environmental footprints of increasingly large data centers.
```

Comment:
Isn't reaching net zero GHG emissions just one goal?

Reply:
The term `net zero goals` is frequent used [1,2,3] to represent a set of owners, contexts, and objectives. 

1. https://www.iea.org/reports/net-zero-by-2050
2. https://www.whitehouse.gov/wp-content/uploads/2021/10/US-Long-Term-Strategy.pdf
3. https://www.worldbank.org/en/news/feature/2022/05/23/what-you-need-to-know-about-net-zero

Reply:
Alright then.

Reply:
Thank you very much for the clarification.